cookbooks/aws-parallelcluster-environment/files/cloudwatch/cloudwatch_agent_config_util.py

```diff
@@ -69,26 +69,14 @@ def _read_json_at(path):
     return None
 
 
-def _read_jinja_template_at(path):
-    """Read the JSON file at path."""
-    try:
-        with open(render_jinja_template(path), encoding="utf-8") as input_file:
-            return json.load(input_file)
-    except FileNotFoundError:
-        _fail(f"No file exists at {path}")
-    except ValueError:
-        _fail(f"File at {path} contains invalid JSON")
-    return None
-
-
 def _read_schema():
     """Read the schema for the CloudWatch log configs file."""
     return _read_json_at(SCHEMA_PATH)
 
 
 def _read_log_configs():
     """Read the current version of the CloudWatch log configs file, cloudwatch_agent_config.json."""
-    return _read_jinja_template_at(LOG_CONFIGS_PATH)
```

Comment:
why removing the render from the Jinja template?

Reply:
because now I am rendering it before call _read_log_configs